zipline/assets/assets.py

```diff
@@ -1532,13 +1502,16 @@ def lifetimes(self, dates, include_start_date, country_codes):
         numpy.putmask
         zipline.pipeline.engine.SimplePipelineEngine._compute_root_mask
         """
-        # normalize this to a cache-key
+        # TODO_SS: Should we just accept a single string here?
+        if isinstance(country_codes, string_types):
```

Comment:
I wrote a very similar thing the other day, string being an iterable is frustrating.